src/test/java/create_task_use_case/CreateTaskInteractorTest.java

```diff
@@ -0,0 +1,19 @@
+package create_task_use_case;
+
+import entities.TaskFactory;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CreateTaskInteractorTest {
+    private final static CreateTaskOutputBoundary createTaskPresenter = new CreateTaskPresenter();
+    private final static CreateTaskDataAccess createTaskDataAccess = new CreateTaskDataAccess();
+    private final static TaskFactory taskFactory = new TaskFactory();
+    @Test
+    public void createTest(){
```

Comment:
As stated in previous comments, you should check if the 'days' attribute of CreateTaskDataAccess is actually mutated, and has a Day instance with a Task instance with title "Finished Unit Test".

Reply:
Agreed and so I have implemented two assertions one to check the key and one to check the value of the tasks hashmap.